Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -317,16 +317,14 @@ open List in
 /-- There are several equivalent ways to say that a number `z` is in fact a real number. -/
 theorem is_real_TFAE (z : K) : TFAE [conj z = z, ∃ r : ℝ, (r : K) = z, ↑(re z) = z, im z = 0] := by
   tfae_have 1 → 4
-  · intro h
+  | h => by
     rw [← @ofReal_inj K, im_eq_conj_sub, h, sub_self, mul_zero, zero_div,
       ofReal_zero]
   tfae_have 4 → 3
-  · intro h
+  | h => by
     conv_rhs => rw [← re_add_im z, h, ofReal_zero, zero_mul, add_zero]
-  tfae_have 3 → 2
-  · exact fun h => ⟨_, h⟩
-  tfae_have 2 → 1
-  · exact fun ⟨r, hr⟩ => hr ▸ conj_ofReal _
+  tfae_have 3 → 2 := fun h => ⟨_, h⟩
+  tfae_have 2 → 1 := fun ⟨r, hr⟩ => hr ▸ conj_ofReal _
```

Comment:
```suggestion
  tfae_have 3 → 2
  | h => ⟨_, h⟩
  tfae_have 2 → 1
  | ⟨r, hr⟩ => hr ▸ conj_ofReal _
```

You could, if you wanted, search-replace (regex) `tfae_have.*:=[\s\n]*fun` with the `| ... =>` pattern. It's also fine if you decide to leave as-is for now.

Reply:
Generally, my policy has been to maintain single-line `fun`s (with term-mode bodies), or match the surrounding style of `tfae_have` if there was one (e.g., all match alts, or all `:=`). 

However, note that another option is keeping things like this single-line with e.g. 
```
tfae_have 3 → 2 | h => ⟨_, h⟩ 
tfae_have 2 → 1 | ⟨r, hr⟩ => hr ▸ conj_ofReal _  
```
It is easy via regex, so if you (or zulip) have a strong opinion on this style vs. `:= fun` style, I'm happy to change it! :)

Reply:
I don't have a strong opinion